code/modules/mob/language.dm

```diff
@@ -570,4 +570,28 @@
 	else
 		return ..()
 
+/datum/language/human/monkey
+	name = "Chimpanzee"
+	desc = "Ook ook ook."
+	speech_verb = "chimpers"
+	ask_verb = "chimpers"
+	exclaim_verb = "screeches"
+	key = "6"
```

Comment:
Technically these keys should be taken by their parent language right? So it's fine to re-use them for monkeys, as no human will speak both the parent language and it's monkey version (hmm although mechs/pAIs might? Or do they just universally understand.